src/System.Runtime/tests/System/TupleTests.cs

```diff
@@ -180,7 +180,7 @@ public void ToString(string expected)
                 Assert.Equal(expected, Tuple.ToString());
             }
 
-            public void Equals_GetHashCode(TupleTestDriver<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10> other, bool expectEqual, bool expectStructuallyEqual)
+            public void Equals_GetHashCode(TupleTestDriver<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10> other, bool expectEqual, bool expectStructuallyEqual, bool netfx)
```

Comment:
Same. I'm not opposed to changing this, but it's orthogonal to this PR. Do you want me to do it here, or in a new PR?